babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -987,3 +987,51 @@ SqlNode AlternativeTypeConversionQuery(SqlNode query) :
         return SqlStdOperatorTable.CAST.createCall(s.end(this), args);
     }
 }
+
+SqlNode RankSortingExpression() :
```

Comment:
Suggest adding a comment stating that the "RANK(foo)" function has a default ordering of DESC and the RANK() OVER ..." clause has a default ordering of ASC.

Reply:
Comment added